index.html

```diff
@@ -1,216 +1,277 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
     <title>Portfolio</title>
 </head>
 <body>
-    <header>
+    <header class="header">
         <div class="head-content">
             <p class="logo">
                <a href="http//">Welcome</a>
             </p>
-            <nav class="nav">
-                <i class="fa-solid fa-bars"></i>
-                <ul class="menu">
-                    <li><a href="#presentation">Portofolio</a></li>
-                    <li><a href="#about">About</a></li>
-                    <li><a href="#contact">Contact</a></li>
-                </ul>
-            </nav>
+            <button class="button" type="button" aria-label="toggle curtain navigation">
+                <span></span>
+            </button>
         </div>
+        <nav class="nav">
```

Comment:
When I click on any of the links in the mobile menu, the navbar should close. Right now it remains open all the time unless I don't click on the close button. Consider fixing it.